### ISSUE TITLE
Add reactions based events

### ISSUE THREAD

**Issue by V-ed** [Vhoyon/Discord-Bot#30]
This could make for some interesting behaviors such as visual confirmation (see example below) or voting!

![example](https://user-images.githubusercontent.com/16652915/38787111-21fa0a8c-40fa-11e8-9ce5-d4408ee9f0f4.png)

_Just some explanations for that example :_ The bot could send a message with default reactions (choices) of which the user would add a reaction (by clicking on the reaction he wants) to input his choice.

In other words, this would enable a new interaction method with the bot, which is always useful.

**Comment by V-ed** [comment]
*TODO* :
Track the message ID of the form in the buffer for the TextChannel and the available options.

A `ReactOption` object could be created to track what option is what and what has been selected. This could be part of a `ReactMessage` where the id of the message would be stored (the first line of this comment).

I imagine something like this :
```java
public class ReactMessage {
	
	public ReactMessage(String messageId, ReactOption... options){
		[...]
	}
	
}
```

and this for the `ReactOption` :
```java
public class ReactOption {
	
	public ReactOption(String emoji, int countBeforeChosen){
		[...]
	}
	
}
```

**Comment by V-ed** [comment]
Issue moved to [Vhoyon/Vramework #2](https://github.com/Vhoyon/Vramework/issues/2) via [**ZenHub**](https://www.zenhub.com/)
